setup.cfg

```diff
@@ -21,6 +21,7 @@ flake8-ignore =
 log_level = WARNING
 markers =
     installation: test requires installation of dependencies
+    functional: Run functional tests using real data
```

Comment:
Maybe calling this marker something like `use_sample_data` would be more clear than `functional`?